app/validators/created_in_past_validator.rb

```diff
@@ -20,6 +20,6 @@ def validate_each(record, attribute, value)
   def validate_date(record, attribute, value, prefix = nil)
     prefix &&= "#{prefix} "
     record.errors.add(attribute, "#{prefix}must have a four digit year") if Settings.earliest_year > value.year
-    record.errors.add(attribute, "#{prefix}must be in the past") if Time.zone.today.year < value.year
+    record.errors.add(attribute, "#{prefix}must be in the past") if Time.zone.today < value
```

Comment:
What is `value` here? Is it a string, a date, an edtf, etc.?

Reply:
It is a Date